manifests/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -82,7 +82,7 @@ template: |
   - name: enable-core-dump
     args:
     - -c
-    - sysctl -w kernel.core_pattern=/var/lib/istio/core.proxy && ulimit -c unlimited
+    - sysctl -w kernel.core_pattern=/var/lib/istio/data/core.proxy && ulimit -c unlimited
```

Comment:
data ihere does not meet linux FHS standards, such as they are.

Reply:
/var/lib should contain persistent data, related to the app.
Do you suggest a different one? I think we should get this in like this, it is in 1.6.